docs/rpc/http/getMultipleAccounts.mdx

```diff
@@ -93,7 +93,7 @@ The result will be a JSON object with `value` equal to an array of:
     \(and is strictly read-only\)
   - `rentEpoch: <u64>` - the epoch at which this account will next owe rent, as
     u64
-  - `space: <u64>` - the data size of the account
+  - `size: <u64>` - the data size of the account
```

Comment:
This one is being changed from `space` to `size`, while the rest are `size` to `space`. Is this intentional?